includes/islandora_scan_batch.inc

```diff
@@ -528,4 +542,12 @@ EOXML;
   protected function addContentModelRelationships() {
     $this->models = $this->preprocessorParameters['content_models'];
   }
+
+  /**
+   * Accessor for content models.
+   */
+  public function getContentModels() {
```

Comment:
Fair point, or should $models be a protected variable. I was trying to mimic the use in Islandora_importer so perhaps IslandoraScanBatchObject should have something like this?
https://github.com/Islandora/islandora_importer/blob/7.x/includes/importer.inc#L224-L228